recipes/torchhd/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "torch-hd" %}
+{% set version = "2.2.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 4ffb7c627441aa71a3e286cab035bbe9c52d7ccfaf2605c613a22cdfea1eeccb
+
+build:
+  number: 0
+  skip: true  # [win]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - numpy
+    - pandas
+    - python
+    - requests
+    - pytorch
+
+test:
+  imports:
+    - torchhd
+    - torchhd.datasets
+
+about:
+  home: "https://github.com/hyperdimensional-computing/torchhd"
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
```

Comment:
The `LICENCE.txt` in this PR actually isn't doing anything, as this line is (correctly) picking up the file `LICENSE` from the package sources

Reply:
Although I just noticed the LICENSE packaged with the pypi [and repository](https://github.com/hyperdimensional-computing/torchhd/blob/f414c9efb17683eccdfb36f3d3de63330aaa717e/LICENSE) is different, as-in, different names under copyright. Do you know that this LICENCE.txt is actually more accurate and this is being corrected upstream? (in which case, this should be changed to point to the internal `LICENSE.txt` so that it isn't ignored)

Reply:
I have updated both the meta.yaml file to point to the correct LICENSE.txt and updated the license text file to be in line with the copyright on the repository as well.